### ISSUE TITLE
Mobile Mockup Pages Candidate

### ISSUE THREAD

**Issue by ecada** [SWE574-Nerds/friendly-eureka#13]
Hello,
I've created candidate Android Mock-Up pages. After the Requirements Elicitaition Web Annotation part will be added. 
![page 1](https://user-images.githubusercontent.com/18642218/31063893-e11fea26-a73f-11e7-8d35-6bb3db628792.png)
![page 2](https://user-images.githubusercontent.com/18642218/31063894-e1215f0a-a73f-11e7-9594-1a57b897ed7a.png)
![page 3](https://user-images.githubusercontent.com/18642218/31063897-e15c5308-a73f-11e7-9708-b73ad9da4abd.png)
![page 4](https://user-images.githubusercontent.com/18642218/31063896-e1254de0-a73f-11e7-9b57-88dc9301b7b1.png)

I look forward for your feedback!

**Comment by muharremunver** [comment]
Is it possible to color the screens? It would be better if we decided on the colors.

**Comment by onatbas** [comment]
This looks good! I'd definitely accept look and feel such as this. Like Muharrem, i too believe color will make for much better but I think there's no color because the tool doesn't allow it (just like balsamiq), and this is a mockup :)

Kudos to you

**Comment by SelinGungor** [comment]
Looks very good. You're doing beautifully.

Additionally, we may also add the same pictures here to increase the virtual customer experience that we use in Web App mockup. I believe mockups are the showcases of our product in an attractive way.
We may also add search page for mobile app as well.

**Comment by ecada** [comment]
Hello, 
At first I didn't have an account in NinjaMockup so when I tried to add Tombili cat's image to the image areas they automatically directed me to the sign-up, log-in page and all my work was lost so I created these mock-ups twice!  Now since I've created an account I was able to add images to the project. Here is the updated version. I hope you'll like them:

![page 1](https://user-images.githubusercontent.com/18642218/31081686-53a4ab04-a795-11e7-8ffb-b8eed434225a.png)
![page 2](https://user-images.githubusercontent.com/18642218/31081685-53a00c8e-a795-11e7-9f7c-9c87b0c6cba6.png)
![page 3](https://user-images.githubusercontent.com/18642218/31081687-53aa4d2a-a795-11e7-85be-79f2ba90dc2c.png)
![page 4](https://user-images.githubusercontent.com/18642218/31081688-53d32b78-a795-11e7-907f-bda09f1cfcc8.png)
